packages/anoncreds-rs/src/services/AnonCredsRsHolderService.ts

```diff
@@ -48,31 +51,35 @@ export class AnonCredsRsHolderService implements AnonCredsHolderService {
     agentContext: AgentContext,
     options?: CreateLinkSecretOptions
   ): Promise<CreateLinkSecretReturn> {
+    let masterSecret: MasterSecret | undefined
     try {
+      masterSecret = MasterSecret.create()
+
+      // FIXME: This is a very specific format of anoncreds-rs. I think it should be simply a string
+      const linkSecretJson = masterSecret.toJson() as { value: { ms: string } }
```

Comment:
Yeah I think it was done like that for Indy-sdk reasons. Can you create an issue for that if we should fix this in anoncreds-rs?

Reply:
Done! https://github.com/hyperledger/anoncreds-rs/issues/138